.github/workflows/lint.yml

```
@@ -21,7 +21,7 @@ jobs:
           skip-node: true
 
       - name: Run Prettier
-        run: bin/bundle exec rbprettier --check --ignore-unknown --loglevel warn '**/*'
+        run: npx prettier --check --ignore-unknown --loglevel warn '**/*'
```

Let's use the yarn version of this instead of relying on `npx`, which is part of `npm`:

```suggestion
        run: yarn run prettier --check --ignore-unknown --loglevel warn '**/*'
```